realm/realm-library/src/main/java/io/realm/internal/TableQuery.java

```diff
@@ -768,9 +771,9 @@ private void throwImmutable() {
 
     private native Long nativeMinimumTimestamp(long nativeQueryPtr, long columnIndex, long start, long end, long limit);
 
-    private native void nativeIsNull(long nativePtr, long[] columnIndices);
+    private native void nativeIsNull(long nativePtr, long[] columnIndices, long[] tablePtrs);
 
-    private native void nativeIsNotNull(long nativePtr, long[] columnIndices);
+    private native void nativeIsNotNull(long nativePtr, long[] columnIndice, long[] tablePtrss);
```

Comment:
Nice catch.  I still have a habit, from eclipse days, of hitting splat-s, to save the file.